tests/console/yast2_lan_device_settings.pm

```diff
@@ -24,8 +24,10 @@ use version_utils qw(is_sle is_leap is_tumbleweed);
 use y2lan_restart_common;
 
 sub run {
-    my $self      = shift;
-    my $static_ip = "192.168.1.119";
+    my $self               = shift;
+    my $static_ip          = "192.168.1.119";
+    my $static_hostname    = 'testhost';
+    my $is_set_in_etc_host = sub { return !script_run('grep ' . shift . ' /etc/hosts') };
```

Comment:
I would propose to use something like `return script_run('grep ' . shift . ' /etc/hosts') != 0` as I find it more readable, but that's more like matter of taste.